tests/virt/node/high_performance_vm/test_high_performance_templates.py

```
@@ -132,8 +132,10 @@ def test_rhel_iothread_pin(self, high_performance_vm):
         ]
     )
     @pytest.mark.polarion("CNV-6758")
-    def test_rhel_change_cpu_core_count(self, high_performance_vm, increased_high_performance_vm_core_count_by_one):
-        assert check_vcpupin_count(high_performance_vm)
+    def test_rhel_change_cpu_core_count(
+        self, high_performance_vm, increased_high_performance_vm_core_count_by_one, admin_client
```

Ruff (0.15.0)

[warning] Unused method argument: `increased_high_performance_vm_core_count_by_one`

(ARG002)